package.json

```
@@ -1,40 +1,33 @@
 {
-  "name": "byu-jwt",
-  "version": "3.1.2",
-  "description": "The byu-jwt module provides helpful functions to retrieve a specified BYU .well-known URL and verify BYU signed JWTs.",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/byu-oit/byu-jwt-nodejs.git"
-  },
-  "author": "Gary Crye",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/byu-oit/byu-jwt-nodejs/issues"
+  "name": "root",
+  "private": true,
+  "engines": {
+    "npm": ">=8",
```

```suggestion
    "npm": ">=9",
```

Not that this actually matters, but the newest LTS versions of Node.js 18 ship with npm 9. I believe that's what we're setting out to support.

---

Fixed in 38e97afeb597df54da8e2bc3cd755722c315d601.